riff-raff/app/persistence/postgresdb.scala

```diff
@@ -43,47 +43,9 @@ class PostgresDatastore(config: Config) extends DataStore(config) with Logging {
         DeployRecordDocument
       ),
       LogDocument.tableName -> getCollectionStats(LogDocument),
-      AuthorisationRecord.tableName -> getCollectionStats(AuthorisationRecord),
       ApiKey.tableName -> getCollectionStats(ApiKey)
     )
 
-  // Table: auth(email: String, content: jsonb)
-  def getAuthorisation(
-      email: String
-  ): Either[Throwable, Option[AuthorisationRecord]] =
-    logExceptions(Some(s"Requesting authorisation object for $email")) {
-      DB readOnly { implicit session =>
-        sql"SELECT content FROM auth WHERE email = $email"
-          .map(AuthorisationRecord(_))
-          .single()
-          .apply()
-      }
-    }
-
-  def getAuthorisationList: Either[Throwable, List[AuthorisationRecord]] =
-    logExceptions(Some("Requesting list of authorisation objects")) {
-      DB readOnly { implicit session =>
-        sql"SELECT content FROM auth".map(AuthorisationRecord(_)).list().apply()
-      }
-    }
-
-  def setAuthorisation(auth: AuthorisationRecord): Either[Throwable, Unit] =
-    logExceptions(Some(s"Creating auth object $auth")) {
-      DB localTx { implicit session =>
-        val json = Json.toJson(auth).toString()
-        sql"INSERT INTO auth (email, content) VALUES (${auth.email}, $json::jsonb) ON CONFLICT (email) DO UPDATE SET content = $json::jsonb"
```

Comment:
Should we drop this table?

Reply:
Yes definitely, let's have a dedicated PR for that as a follow-up to this one.